src/DataList/DataList.js

```diff
@@ -68,7 +68,10 @@ var DataListSpotlightSupport = {
 				child = this.childForIndex(inIndex);
 			}
 			subChild = inSubChild ? Spotlight.getChildren(child)[inSubChild] : child;
-			Spotlight.spot(subChild);
+			if (Spotlight.isSpottable(subChild)) 
```

Comment:
This logic here is sound. Add `{}` around the `if`/`else` blocks.

An alternative, though perhaps not any better, would be to inline the two `spot()` commands. Since `spot()` calls `isSpottable()` internally and will return false if not, you can do the following to the same effect:

```
Spotlight.spot(subChild) || Spotlight.spot(this);
```